App.js

```diff
@@ -1,36 +1,71 @@
 import React, { useEffect, useState } from "react";
 import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
-// import { onAuthStateChanged, signOut } from "firebase/auth";
-// import { auth } from "./Firebase/firebaseSetup";
-// import { Button } from "react-native";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 import Me from "./Components/Me";
 import MeEdit from "./Components/MeEdit";
+import Posts from "./screens/Posts";
+import AddPost from "./screens/AddPost";
+import { Button, Icon } from "@rneui/base";
 // import AntDesign from "@expo/vector-icons/AntDesign";
 
+const Stack = createStackNavigator();
+const Tab = createBottomTabNavigator();
+
 export default function App() {
   // TODO:need auth
   const isUserLoggedIn = true;
-  const Stack = createStackNavigator();
-
   const AuthStack = (
     <>
       {/*<Stack.Screen name="Login" component={Login} />*/}
       {/*<Stack.Screen name="Signup" component={Signup} />*/}
     </>
   );
 
-  const AppStack = (
-    <>
-      <Stack.Screen name="Me" component={Me} />
-      <Stack.Screen name="MeEdit" component={MeEdit} />
-    </>
-  );
+  const Tabs = () => {
+    return (
+      <Tab.Navigator
+        screenOptions={({ navigation, route }) => ({
+          // Adding a custom AddButton in the header for Activities and Diets, not for Settings
+          headerRight: () => {
+            return route.name !== "Settings" ? (
+              <Button
+                title="Add Post"
+                onPress={() => navigation.navigate("AddPost")}
+              />
+            ) : null;
+          },
+        })}
+      >
+        <Tab.Screen name="Posts" component={Posts} />
+        <Tab.Screen name="Me" component={Me} />
+      </Tab.Navigator>
+    );
+  };
 
   return (
     <NavigationContainer>
       <Stack.Navigator>
-        {isUserLoggedIn ? AppStack : AuthStack}
```

Comment:
use appStack or AuthStack for iteration 2 adding authentication

Reply:
Solved. Thanks for notice.